eth/tracers/logger/logger_test.go

```diff
@@ -46,24 +47,29 @@ type dummyStatedb struct {
 	state.StateDB
 }
 
-func (*dummyStatedb) GetRefund() uint64 { return 1337 }
+func (*dummyStatedb) GetRefund() uint64                                       { return 1337 }
+func (*dummyStatedb) GetState(_ common.Address, _ common.Hash) common.Hash    { return common.Hash{} }
+func (*dummyStatedb) SetState(_ common.Address, _ common.Hash, _ common.Hash) {}
 
 func TestStoreCapture(t *testing.T) {
+	logger := NewStructLogger(nil)
 	var (
-		env      = NewEVM(BlockContext{}, TxContext{}, &dummyStatedb{}, params.TestChainConfig, Config{})
-		logger   = NewStructLogger(nil)
-		contract = NewContract(&dummyContractRef{}, &dummyContractRef{}, new(big.Int), 0)
-		scope    = &ScopeContext{
-			Memory:   NewMemory(),
-			Stack:    newstack(),
-			Contract: contract,
-		}
+		env      = vm.NewEVM(vm.BlockContext{}, vm.TxContext{}, &dummyStatedb{}, params.TestChainConfig, vm.Config{Debug: true, Tracer: logger})
+		contract = vm.NewContract(&dummyContractRef{}, &dummyContractRef{}, new(big.Int), 100000)
 	)
-	scope.Stack.push(uint256.NewInt(1))
-	scope.Stack.push(new(uint256.Int))
+	contract.Code = []byte{byte(vm.PUSH32)}
+	one := uint256.NewInt(1).Bytes32()
+	zero := new(uint256.Int).Bytes32()
+	contract.Code = append(contract.Code, one[:]...)
+	contract.Code = append(contract.Code, byte(vm.PUSH32))
+	contract.Code = append(contract.Code, zero[:]...)
+	contract.Code = append(contract.Code, byte(vm.SSTORE))
```

Comment:
```suggestion
	contract.Code = []byte{
		byte(vm.PUSH1), 0x1, 
		byte(vm.PUSH1), 0x0, 
		byte(vm.SSTORE),
	}
```

Reply:
Could even be further simplified
```
	contract.Code = []byte{
		byte(vm.PUSH1), 0x1, byte(vm.DUP1), byte(vm.SSTORE),
	}
```

Reply:
Ah right, that's what I had first but it was failing, as I later found out, for another reason. Fixed